web/themes/custom/itkdev/itkdev_project_theme/js/disableAutoEvaluation.js

```diff
@@ -0,0 +1,13 @@
+let checkbox = document.getElementById('edit-project-track-evaluation-set-manual')
+let autoEvaluation = document.getElementById('auto-evaluation')
+
+checkbox.addEventListener("change", function() {
+  autoEvaluation.classList.toggle('text-slate-300')
+})
+
+if (checkbox.checked) {
+  autoEvaluation.classList.add('text-slate-300')
+}
+else {
+  autoEvaluation.classList.remove('text-slate-300')
+}
```

Comment:
Consider adding a helper function to make it a little more clear that the final `if` statement intializes the state:

```suggestion
const updateState () => autoEvaluation.classList.toggle('text-slate-300', checkbox.checked)
checkbox.addEventListener("change", updateState)
// Initialize.
updateState()
```